tests/cases/fourslash/jsDocFunctionSignatures3.ts

```diff
@@ -0,0 +1,32 @@
+///<reference path="fourslash.ts" />
+
+// @allowNonTsExtensions: true
+// @Filename: Foo.js
+
+//// var someObject = {
+////     /**
+////      * @param {string} param1 Some string param.
+////      * @param {number} parm2  Some number param.
+////      */
+////     someMethod: function(param1, param2) {
+////         console.log(param1/*1*/);
+////         return false;
+////     },
+////     /**
+////      * @param {number} p1  Some number param.
+////      */
+////     otherMethod(p1) {
+////         p1/*2*/
+////     }
+////     
+//// };
+
+goTo.marker('1');
+edit.insert('.');
+verify.memberListContains('substr', undefined, undefined, 'method');
+edit.backspace();
+
+goTo.marker('2');
+edit.insert('.');
+verify.memberListContains('toFixed', undefined, undefined, 'method');

```

Comment:
Nit: instead of checking the member list which is dependent on the apparent type (and which can potentially be affected by various usages of identifiers), I think it's generally a better idea to stick with the quick info since that will tell you precisely the type you're using.